### PR TITLE
Bugfix/warning qa

### PR DESCRIPTION
**Pull request by piyalbasu**
There was a couple of small issues with these warning modals.

1) The LoadingBackground that covers the screen behind the modal wasn't covering the entire screen. Now using a portal to make sure we're not constrained by the DOM hierarchy

2) There were a couple of references to the old color var's and hex values. Replacing these with the SDS v3 versions for consistency

3) removing some unused modals and styles for cleanliness 

4) Fixed a legacy issue where the styling for the New Asset warning modal was a little different than the Scam Asset warning modal. There's still some work to be done here, but at the very least, they're not wildly different anymore. We'll need some design input on this

### CODE REVIEW

**Comment by piyalbasu** [comment]
Screenshot 2024-11-11 at 2 40 45 PM
Screenshot 2024-11-11 at 2 55 10 PM

**Comment by piyalbasu** [comment]
Screenshot 2024-11-11 at 2 59 50 PM
Screenshot 2024-11-11 at 2 55 10 PM

File: extension/src/popup/components/WarningMessages/index.tsx [hunk 1]
```diff
@@ -1079,18 +1079,6 @@ export const BlockAidMaliciousLabel = () => {
   );
 };
 
-export const BlockAidBenignLabel = () => {
```
Comment (piyalbasu) [inline extension/src/popup/components/WarningMessages/index.tsx:1082]: we don't use this bengin label, so removing the component along with its styles

File: extension/src/popup/components/ModalInfo/index.tsx [hunk 2]
```diff
@@ -75,7 +75,7 @@ export const ModalInfo = ({
 
   return (
     <div className={cardClasses}>
-      <Card variant="secondary">
+      <Card>
```
Comment (piyalbasu) [inline extension/src/popup/components/ModalInfo/index.tsx:78]: in the new design system, we should use the `primary` card, which has just a _little_ different bg color

File: extension/src/popup/components/WarningMessages/styles.scss [hunk 3]
```diff
@@ -211,19 +212,6 @@
...
-      --Button-color-background: var(--sds-clr-red-06);
-      --Button-color-border: var(--sds-clr-red-09);
-      background: var(--Button-color-background);
-      border-color: var(--Button-color-border);
-    }
-
-    .BasicButton--primary:hover {
-      border-color: var(--sds-clr-red-09);
-      background-color: var(var(--sds-clr-red-07));
-    }
```
Comment (piyalbasu) [inline extension/src/popup/components/WarningMessages/styles.scss:226]: these have been replaced with the new SDS error button styles

**Review by aristidesstaffieri — APPROVED** [review]

**Pull request merged**